git-sidecar/rootfs/clone.sh

```diff
@@ -6,6 +6,10 @@ set -x
 : "${VCS_REPO:?}"
 : "${VCS_REVISION:?}"
 
+if sha=$(git ls-remote --exit-code "${VCS_REPO}" "${VCS_REVISION}" | cut -f1); then
```

Comment:
Nit but shouldn't VCS_REVISION be renamed to e.g. `VCS_REF` as it doesn't stand only for REVISION now?

Reply:
I have another PR that fixing the naming